_posts/2017-07-05-SpringIO17-Summary.md

```diff
@@ -0,0 +1,773 @@
+---
+layout: post
+authors: [dieter_hubau, gina_de_beukelaer, hans_michiels, jeff_mesens]
+title: 'Spring I/O 2017 Recap'
+image: /img/springio2017.jpg
+tags: [Spring, IO, Pivotal, Google, Spring Boot, Java, Reactive, Reactor, DDD, CQRS, Spring Cloud, Spring Cloud Stream, Serverless]
+category: Spring
+comments: true
+---
+
+> On the 18th and 19th of May, we had another great edition of Spring I/O, brought to us by organizer [Sergi Almar](https://twitter.com/sergialmar){:target="_blank"}.
+In this blog post, we will go over some of our favourite sessions of the conference.
+
+> **DISCLAIMER**: this is obviously a very opinionated list of talks. This does not mean that the other talks were of lower quality, it just means we were not able to attend or did not take extensive notes.
+
+# Keynote: The Only Constant Is Change
+
+### by [Andy Wilkinson](https://twitter.com/ankinson){:target="_blank"} and [Stéphane Nicoll](https://twitter.com/snicoll){:target="_blank"}
+
+Obviously, the keynote presentation was filled with announcements, interesting tidbits and two great presenters.
+
+The biggest topic revolved around how **Spring has always tried to enable developers to adapt their applications rapidly**.
+This capacity of adapting to change increased dramatically when Spring Boot was released, which explains part of its success.
+
+They also reported the release of [Spring Boot 2.0.0 M1](https://spring.io/blog/2017/05/16/spring-boot-2-0-0-m1-available-now).
+This release was announced very shortly after [Spring Framework 5.0 went RC1](https://spring.io/blog/2017/05/08/spring-framework-5-0-goes-rc1) on the 8th of May.
+
+The keynote can be found [here](https://www.youtube.com/watch?v=uJSYnqIhcIo){:target="_blank"} and the demo code [here](https://github.com/snicoll-demos/demo-webflux-streaming){:target="_blank"}.
+
+# Bootiful Database-centric applications with JOOQ
```

Comment:
Although titles are capitalised, replace JOOQ with jOOQ. Also for all other occurrences.